CHANGELOG.md

```diff
@@ -41,10 +37,15 @@ All versions prior to 0.9.0 are untracked.
 * **BREAKING API CHANGE**: `VerificationMaterials` has been removed.
   The public verification APIs now accept `sigstore.verify.models.Bundle`.
 
+* **BREAKING API CHANGE**: `Signer.sign(...)` has been removed. Use
+  either `sign_artifact(...)` or `sign_intoto(...)`, depending on whether
+  you're signing opaque bytes or an in-toto statement.
+
 ### Changed
 
-* **BREAKING API CHANGE**: The `Signer.sign(...)` API now returns a `sigstore.verify.models.Bundle`,
-  instead of a `SigningResult` ([#862](https://github.com/sigstore/sigstore-python/pull/862))
+* **BREAKING API CHANGE**: The `Signer.sign_artifact(...)` API now returns
+  a `sigstore.verify.models.Bundle`, instead of a `SigningResult`
+  ([#862](https://github.com/sigstore/sigstore-python/pull/862))
```

Comment:
I suppose this can just be removed since we already mention that `sign_artifact` has been added above.